testing_and_setup/compass/ocean/global_ocean/EC60to30wISC/init/config_E3SM_coupling_files.ini

```diff
@@ -0,0 +1,78 @@
+# This is the custom config file specific to this test case.  Any values here
+# will override the defaults in defaults.ini  Users wishing to alter the
+# configuration should modify this file, rather than defaults.ini
+
+[main]
+nprocs = 1
+# the file name of the initial condition, possibly after spin-up
+initial_condition = ../../spin_up/simulation/restarts/restart.0001-21-01_00.00.00.nc
+
+[mesh]
+short_name = ${prefix}_${min_res}to${max_res}km_${levels}L_E3SMv${e3sm_version}_rev${mesh_version}
+prefix = EC_wISC
+long_name = Eddy Closure mesh with Ice Shelf Cavities for E3SM Version
+            ${e3sm_version}, ${min_res}-${max_res}km resolution,
+            ${levels} vertical levels
+description = MPAS Eddy Closure mesh with enhanced resolution around the equator
+              (30 km) and poles (35 km) with 60-km resolution at mid latitudes
+              and includes cavities under the ice shelves in the Antarctic.
+e3sm_version = 2
+mesh_version = 001
+creation_date = autodetect
+min_res = 30
+max_res = 60
+max_depth = autodetect
+levels = autodetect
+runoff_description = <<<Spreading function described here>>>
+
+# The following options are detected from .gitconfig if not explicitly entered
+author = autodetect
+email = autodetect
```

Comment:
@vanroekel, a copy of this file will end up in the `e3sm_coupling` directory.  You could edit the `email` option here and put in your LANL email address.